src/dateUtils.js

```diff
@@ -0,0 +1,44 @@
+const { DateTime, Duration } = require('luxon');
+
+const toDate = val => {
+  if (DateTime.isDateTime(val)){
```

Comment:
can you add a unit test for each block?
can you make sure the behaviour of the old function and the new function are identical (the same tests pass for both functions)?

Reply:
Done